x-pack/plugins/cases/common/ui/types.ts

```diff
@@ -103,6 +103,15 @@ export interface ResolvedCase {
   aliasPurpose?: ResolvedSimpleSavedObject['alias_purpose'];
 }
 
+export interface Attachment {
+  id: string;
+  fileName: string | null | undefined;
```

Comment:
How can this be undefined?

Reply:
Same as above, these were the initial types for the mock data so not really relevant anymore.